source/jormungandr/jormungandr/scenarios/helper_classes/helper_utils.py

```diff
@@ -766,3 +777,34 @@ def complete_transfer(pt_journey, transfer_pool):
         if section.type != response_pb2.TRANSFER:
             continue
         transfer_pool.wait_and_complete(section)
+
+
+def prepend_first_and_append_last_coord(dp, origin, destination):
```

Comment:
Maybe split this function in two `prepend_first_coord()` and `append_last_coord()` ?

Reply:
And don't you already have the exact same function [here](https://github.com/hove-io/navitia/blob/fca4622c715418b16686de7a106d836ff1fd4179/source/jormungandr/jormungandr/scenarios/helper_classes/streetnetwork_path.py#L104) ?